db/migrate/20170706172353_remove_category_column_from_repositories.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveCategoryColumnFromRepositories < ActiveRecord::Migration
+  def down
```

Comment:
This should be part of 'up' since it will happen when the migration is 'brought up'. Right now it will only run if you `rake db:rollback` once this has been migrated